src/dlgPackageExporter.cpp

```diff
@@ -41,7 +41,9 @@
 #include "post_guard.h"
 
 // We are now using code that won't work with really old versions of libzip:
-#if (LIBZIP_VERSION_MAJOR < 1) && (LIBZIP_VERSION_MINOR < 11)
+// Unfortunately libzip 1.70 forgot to include these defines and thus broke the
+// original tests:
+#if defined(LIBZIP_VERSION_MAJOR) && defined(LIBZIP_VERSION_MAJOR) && (LIBZIP_VERSION_MAJOR < 1) && (LIBZIP_VERSION_MINOR < 11)
```

Comment:
Should `defined(LIBZIP_VERSION_MAJOR)` appear twice?

Reply:
Good catch, fixed it.

Reply:
:blush: Ooops - copy-pasta error on my part.